O/oathtool/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "oathtool"
```

Comment:
Looks like this is called "OATH Toolkit" https://www.nongnu.org/oath-toolkit/:
```suggestion
name = "OATHToolkit"
```
Need to rename also the directory to exactly match this name

Reply:
> Need to rename also the directory to exactly match this name